agent/lib/kontena/network_adapters/weave.rb

```diff
@@ -233,6 +233,16 @@ def post_start(info)
       Celluloid::Notifications.publish('network_adapter:start', info)
     end
 
+    # @param [Docker::Container] weave
+    # @param [Hash] config
+    def config_changed?(weave, config)
+      return true if weave.config['Image'].split(':')[1] != WEAVE_VERSION
+      cmd = Hash[*weave.config['Cmd'].flatten(1)]
```

Comment:
A bit of reverse-engineering also gives this: `$ curl -s http://127.0.0.1:6784/report -H 'Accept: application/json' | vgrep TrustedSubnets`

```
{
    "Version": "1.7.2",
    ...
    "Router": {
        ...
        "TrustedSubnets": [
            "192.168.66.0/24"
        ],
```